shared/features/filters/period-filter/period-filter.tsx

```diff
@@ -82,26 +82,37 @@ export function PeriodFilter({ onChange, value, dateRangeType, size = "sm" }: Pe
           </div>
         )}
       </Popover.Trigger>
-      <Popover.Content>
+      <Popover.Content className="!px-0 !py-0">
```

Comment:
Maybe add a prop to handle this ?

Reply:
don't know it's the only case we want to do this